gam_app/management/commands/importar_una_bolsa.py

```diff
@@ -235,6 +235,10 @@ def handle(self, *args, **options):
         bolsas_nuevas = bolsas.difference(bolsas_importadas)
         bolsas_nuevas = list(bolsas_nuevas)
 
+        if len(bolsas_nuevas) == 0:
```

Comment:
Not a big thing but it's a bit more idiomatic to write `if bolsas_nuevas` then `if len(bolsas_nuevas) == 0`

Reply:
Wouldn't `if bolsas_nuevas` check if the variable is true or false?  It does exist, so it would return true.  However, I want to check the number of items in the list.

Reply:
`if whatever` checks if `bool(whatever)` evaluates to true or false. `bool` is defined for lists as false when the list is empty and true otherwise, which is why that idiom works. If the variable didn't exist at all it would just be a regular `NameError` exception.

Reply:
Thank you!  Great thing to know.